tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -38,6 +38,70 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( "[Blocking] int Hello() => 4;", actual.Value.ToFullString() );
 	}
 
+	[Test]

```

Comment:
![image](https://user-images.githubusercontent.com/597907/153487052-4c4d8386-e817-4494-bd02-f0420fd30433.png)

When we support actual syncification I'll change this test to include `await`s nested in every awful place.